src/app/record/components/work/work.component.html

```diff
@@ -15,6 +15,28 @@
     <app-display-external-ids
       [externalIdentifiers]="work.workExternalIdentifiers"
     ></app-display-external-ids>
+    <div class="general-data" *ngIf="work?.contributors?.length > 0">
+      <span i18n="#upperCase@@works.contributors"> Contributors </span>:
```

Comment:
the `upperCase` will make `Contributors` be displayed as  `CONTRIBUTORS` on QA, is this what we want?

Reply:
We can remove this to avoid it been show in UPPER_CASE

Reply:
Sorry Dani! Looking at the figma file 
![image](https://user-images.githubusercontent.com/2119626/140827578-83f87437-ae79-4b67-b3b1-a1c21c56b54a.png)
We DO need this to be upper case